src/main/java/src/Product.java

```diff
@@ -1,128 +1,121 @@
 package src;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 
 public class Product implements java.io.Serializable {
```

Comment:
I don't think you need to implements the Serializable interface.

Reply:
Fixed.